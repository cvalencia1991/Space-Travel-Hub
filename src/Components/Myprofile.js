import React from 'react';
import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import Col from 'react-bootstrap/Col';

export default function Myprofile() {
  return (
    <Container>
      <Row>
        <Col>My Missions</Col>
        <Col>My Rockets</Col>
      </Row>
    </Container>
  );
}
