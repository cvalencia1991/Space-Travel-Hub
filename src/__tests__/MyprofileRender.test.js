import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Myprofile from '../Components/Myprofile';
import store from '../Redux/configureStore';

it('renders correctly', () => {
  const myProfile = renderer
    .create(<Provider store={store}><Myprofile /></Provider>)
    .toJSON();
  expect(myProfile).toMatchSnapshot();
});
