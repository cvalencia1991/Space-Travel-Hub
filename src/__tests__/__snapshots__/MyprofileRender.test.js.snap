// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h2>
        My Rockets
      </h2>
      <div
        className="mt-4 list-group"
        onKeyDown={[Function]}
      />
    </div>
    <div
      className="col"
    >
      <h2>
        My Missions
      </h2>
      <div
        className="mt-4 list-group"
        onKeyDown={[Function]}
      />
    </div>
  </div>
</div>
`;
